internal/modules/springboot2/charts.go

```diff
@@ -0,0 +1,19 @@
+package springboot2
+
+import (
+	"github.com/l2isbad/go.d.plugin/internal/pkg/charts/raw"
+)
+
+var charts = raw.Charts{
```

Comment:
I suggest to use aliases.
```go
type (
	Charts      = raw.Charts
	Order       = raw.Order
	Definitions = raw.Definitions
	Chart       = raw.Chart
	Options     = raw.Options
	Dimensions  = raw.Dimensions
	Dimension   = raw.Dimension
)
```

Much cleaner

Reply:
I used to use aliases, but `go lint` complains write comment :(

Reply:
tune it then :smile: , **aliases are nice**

Reply:
or ignore in that particular case

Reply:
I wonder why should we need both `raw` and `cooked` version?

Reply:
> if it will looks bad - no go (like map[string]*Chart, even if it makes sense)

I agree.

> ignore in that particular case

The green wavy line (go lint) is very harmful for my eye and there no way to disable it.

Reply:
I know that feel bro 
Does bunch of `raw.*`'s is not harmful for you? :smiley: Especially if you know that they can be avoided... Do alias, remove raw, save the kittens

Reply:
Just joking. Let's leave it as is. I will suffer, i can handle it.